packages/appkit/src/client/appkit-base-client.ts

```diff
@@ -431,7 +431,17 @@ export abstract class AppKitBaseClient {
 
         StorageUtil.addConnectedNamespace(chainToUse)
         this.syncProvider({ ...res, chainNamespace: chainToUse })
-        const { accounts } = await adapter.getAccounts({ namespace: chainToUse, id })
+        /*
+         * SyncAllAccounts already set the accounts in the state
+         * and its more efficient to use the stored accounts rather than fetching them again
+         */
+        const syncedAccounts = AccountController.state.allAccounts
```

Comment:
could these be outdated? what if namespace changes

Reply:
if on init, shouldn't. Still, wagmi firing some kind of event during init might cause something like this